central/pruning/pruning.go

```diff
@@ -1031,6 +1038,32 @@ func (g *garbageCollectorImpl) pruneLogImbues() {
 	postgres.PruneLogImbues(pruningCtx, g.postgres, logImbueWindow)
 }
 
+func (g *garbageCollectorImpl) pruneOrphanedNodeCVEs() {
+	retentionDays := int64(env.OrphanedCVEsRetentionDurationDays.IntegerSetting())
+	if retentionDays < 0 {
+		log.Errorf("Invalid value of ROX_ORPHANED_CVES_RETENTION_DURATION_DAYS env var: %d, value should be >= 0", retentionDays)
```

Comment:
Warning maybe?  Should we return?  Should we use a default instead in this case?

Reply:
Good catch. I did not return or do anything with it after the error. I think we can fallback to default value in that case.